fpmsyncd/routesync.cpp

```diff
@@ -962,3 +968,36 @@ string RouteSync::getNextHopIf(struct rtnl_route *route_obj)
 
     return result;
 }
+
+/*
+ * Get next hop weights
+ * @arg route_obj     route object
+ *
+ * Return concatenation of interface names: wt0 + "," + wt1 + .... + "," + wtN
+ */
+string RouteSync::getNextHopWt(struct rtnl_route *route_obj)
+{
+    string result = "";
+
+    for (int i = 0; i < rtnl_route_get_nnexthops(route_obj); i++)
+    {
+        struct rtnl_nexthop *nexthop = rtnl_route_nexthop_n(route_obj, i);
+        /* Get the ID of next hop interface */
+        uint8_t if_weight = rtnl_route_nh_get_weight(nexthop);
+        if (if_weight)
+        {
+            result += to_string(if_weight + 1);
```

Comment:
please rename to `weight`. `if_weight `is confusing. Also, why is it incremented by 1?

Reply:
@prsunny Incrementing by 1 is to agree with kernel behavior(ip route show). I'll change if_weight to weight